src/platform/apollolake/apollolake.x.in

```diff
@@ -464,6 +460,11 @@ SECTIONS
     KEEP (*(.fw_ready_metadata))
   } >sof_fw :sof_fw_phdr
 
+  .static_uuid_entries : ALIGN(4)
```

Comment:
Is this the same change that @ktrzcinx has just submitted ? Can you guys align.

Reply:
No, this one is about `.static_uuid_entries`, I've made PR for `.trace_ctx` section.

Reply:
ok, so we have any dependencies for merging ?

Reply:
I don't think so